extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -256,7 +257,7 @@ static public function getDomains( $city_id, $master = false ) {
 	static public function addDomain( $city_id, $domain, $reason = '' ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 6, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)